### DISCUSSION TITLE
Use a base_URL or a slash instead of relative path for internal resources

### DISCUSSION THREAD

**Discussion by JoeTessier** [squidfunk/mkdocs-material#4660] [Q&A] [answered]
Hi @squidfunk thx for your job ! 🙏

I was wondering if it would be possible to use a global variable for the `base_url` or a regular `/` character at the start of a link to tell to fetch the resource from the root ? 

ex/ 
❌ instead of having `![My image](../../../assets/img/folder/image.png)`, 
 
just being able to write 
✅ `![My image](/assets/img/folder/image.png)` or
✅ `![My image]({{ base_URL }}/assets/img/folder/image.png)` 

Or maybe there's already a `baseURL` data somewhere, but the best option imo is to start with a slash, and it doesn't seem working atm. 

Thanks for you help, 
Have a nice evening ✌

**Comment by squidfunk** [comment]
This is essentially a MkDocs question, as it's not specifically related to Material for MkDocs. As I understand, there are several plugins for managing URLs, some of which allow for absolute URLs. Check out the [MkDocs plugin wiki](https://github.com/mkdocs/mkdocs/wiki/MkDocs-Plugins#links--references).

**Reply by JoeTessier** [reply]
Copy that, fine by me. Thanks 🙏

**Comment by gledos** [comment]
i use [mkdocs-abs-rel-plugin](https://github.com/sander76/mkdocs-abs-rel-plugin), it's nice plugin.

But after using this plugin, you can only use absolute link.

**Comment by kamilkrzyskow** [comment, marked as answer]
With version 1.6.0 `relative_to_docs` value for `validation` was introduced, which allows to use the absolute paths starting with `/` as described in the OP post:

```yaml
validation:
  absolute_links: relative_to_docs
```

https://www.mkdocs.org/about/release-notes/#validation
https://www.mkdocs.org/user-guide/configuration/#validation
